config_files/default_input_parameters/grd.yaml

```diff
@@ -23,3 +23,8 @@ base_voltage : 416 # base voltage for per unit system in V
 subset_line_losses_modelled : 30 # Number of lines accounted for in losses
 tol_rel_voltage_diff: 1e-2
 tol_rel_voltage_costs: 1e-2
+reactive_power_for_voltage_control: False
+pf_non_flex_heat_home_car : 0.95 # Power factor used for passive houses, heat and car power
+pf_flexible_heat_home : 0.95 # Power factor used for flexible houses for home and heat power
+max_active_power_car : 6.6 # Max active power delivered by and to the car battery, convetional charger
+max_apparent_power_car: 7.0 # Max apparent power delivered by and to the car battery, convetional charger
```

Comment:
could you please add references for `max_active_power_car` and `max_apparent_power_car`?

Reply:
Do you mean adding a link? Is that what you also meant in #10 ? :)

Reply:
either a link or at least the name of the place where you took these numbers from e.g. academic publication, car specifications, etc. Yes, I meant this in #10, finding trustworthy sources and adding the specifications.

Reply:
Because I think that at the time I had taken Tesla specifications, but I do not mind changing if you have data for another model. We just need to say where we took the numbers from and write it down. But please if you change any default values in the parameters let me know.

Reply:
I've noticed that in the grid settings, we used to use `V: 415 # V nominal voltage from Mortsyn 2019 multiclass`. Do you mind changing this value to 416, which corresponds to what the IEEE low voltage test feeder uses?

Reply:
I've now added more details about the power, power factors and voltages used.

Reply:
> I've noticed that in the grid settings, we used to use `V: 415 # V nominal voltage from Mortsyn 2019 multiclass`. Do you mind changing this value to 416, which corresponds to what the IEEE low voltage test feeder uses?

Which equations do you use this parameter in?

Reply:
> I've now added more details about the power, power factors and voltages used.

great thanks

Reply:
> > I've noticed that in the grid settings, we used to use `V: 415 # V nominal voltage from Mortsyn 2019 multiclass`. Do you mind changing this value to 416, which corresponds to what the IEEE low voltage test feeder uses?
> 
> Which equations do you use this parameter in?

I've never used. I though you had implemented it to calculate the upsteam losses? It's only for consistency.

Reply:
ok, if it is just for consistency I think it is best that I stick with this value, as this is consistent with the papers where the methodology for getting upstream losses is laid out, and with the value for the average resistance in the upstream grid.

Reply:
Yes, I see. But then it is not perfectly consistent with the grid we're modelling, which has 416 V. I guess this is not a big issue.

Reply:
yes I agree this is slightly frustrating but the change is very low, and then at least we are self consistent with the calculations for upstream losses. Thanks for pointing this out!

Reply:
Okay. I will close issue #10 and this thread :)

Reply:
it looks like the ref gives the battery specifications of 6.6 kVA so apparent power, not active power? also, does it say the max charge and discharge rates are the same? and it would be nice to have specs for one battery that also include its maximum capacity in kWh if we can find that. If we can't then fair enough but it would have been neat to have all the specs from one car.

Reply:
also --car specs should be in `car.yaml`, not `grd.yaml`